src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationIntTest.kt

```diff
@@ -111,7 +111,7 @@ class PrisonPersonMigrationIntTest : SqsIntegrationTestBase() {
       @Test
       fun `will put message on DLQ if call to NOMIS fails`() {
         nomisApi.stubGetPrisonIds(totalElements = 1, pageSize = 10, firstOffenderNo = "A0001KT")
-        physicalAttributesNomisApi.stubGetPhysicalAttributes(INTERNAL_SERVER_ERROR)
+        physicalAttributesNomisApi.stubGetPhysicalAttributes("A0001KT", INTERNAL_SERVER_ERROR)
```

Comment:
There's a fair bit of this kind of refactor to cater for the fact that move booking has 2 offenders in scope so we can't create stubs againstoffenderNo=.* anymore.